.cirrus.yml

```diff
@@ -42,5 +48,32 @@ release_task:
     - python3.7 -m ensurepip
     - python3.8 -m ensurepip
   build_script: ./make-nix-release.sh
+  script: |
+    #!/usr/bin/env bash
+
+    if [[ "$CIRRUS_RELEASE" == "" ]]; then
+      echo "Not a release. No need to deploy!"
+      exit 0
+    fi
+
+    if [[ "$GITHUB_TOKEN" == "" ]]; then
+      echo "Please provide GitHub access token via GITHUB_TOKEN environment variable!"
+      exit 1
+    fi
+
+    file_content_type="application/octet-stream"
+    files_to_upload=(releases/*)
```

Comment:
This won't work on FreeBSD -- unless they're shipping `bash` by default in the image, which would be odd. Assuming only POSIX `sh` is available, I would just change line 67 to `for fpath in releases*`.